recipes/azure-storage-cpp/all/patches/0001-cmake-fixes.patch

```diff
@@ -0,0 +1,72 @@
+diff --git a/Microsoft.WindowsAzure.Storage/CMakeLists.txt b/Microsoft.WindowsAzure.Storage/CMakeLists.txt
+index ac9e65d..f5988f6 100644
+--- a/Microsoft.WindowsAzure.Storage/CMakeLists.txt
++++ b/Microsoft.WindowsAzure.Storage/CMakeLists.txt
+@@ -46,9 +46,9 @@ if(UNIX)
+   find_package(OpenSSL 1.0.0 REQUIRED)
+ 
+ 
+-  find_package(UUID REQUIRED)
+-  find_package(Casablanca REQUIRED)
+-  find_package(LibXML2 REQUIRED)
++  find_package(libuuid REQUIRED)
++  find_package(cpprestsdk REQUIRED)
++  find_package(LibXml2 REQUIRED)
```

Comment:
Is the one exposed by conan wrong?

Reply:
no, upstream library uses its own find modules: https://github.com/Azure/azure-storage-cpp/tree/master/Microsoft.WindowsAzure.Storage/cmake/Modules